docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -500,28 +500,28 @@ initially in tree[I].Len; the codes are produced in tree[I].Code.
        code length:
 
 .KS
-          code = 0;
-          bl_count[0] = 0;
-          for (bits = 1; bits <= MAX_BITS; bits++) {
-             code = (code + bl_count[bits-1]) << 1;
-             next_code[bits] = code;
-          }
+         code = 0;
```

Comment:
What I meant is that the indentation relative to the paragraph above ("Find the numerical value ...") should be 3 spaces (and likewise for the for loop below).